bazarr.py

```diff
@@ -50,30 +50,6 @@ def register(self, process):
     def unregister(self, process):
         self.__processes.remove(process)
 
-    @staticmethod
-    def __wait_for_processes(processes, timeout):
```

Comment:
I removed the waiting part.

From what I could grab over the internet it seems that `poll` and `wait` method don't really coexist smoothly.